test/step.test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+const math = require('mathjs');
+
+const stepThroughExpression = require('../lib/simplifyExpression/stepThrough');
+
+function testStepsInExpression(exprStr, outputStr) {
+  it ('test the steps in simplifying expression', function () {
+    assert.equal((stepThroughExpression(math.parse(exprStr))[0]['changeType']), outputStr);
```

Comment:
It works for the examples like `2+2` or `2*3*4`. where we get the changeGroup and changeType and the whole node is tested. But it cannot be used for complex examples.